test/nic.js

```diff
@@ -0,0 +1 @@
+// hihi
```

Comment:
No new line at the end of the file. Can see by the red marker. Usually we need to add an empty line at the end of the file :)!